snc-library.sh

```diff
@@ -46,6 +46,7 @@ function download_oc() {
 }
 
 function run_preflight_checks() {
+	local preset=${1:-snc}
```

Comment:
Might be better to name this "bundle_type" as this is the terminology used in snc?